templates/submission/list.html

```diff
@@ -297,7 +294,17 @@ <h3>{{ _('Filter submissions') }} <i class="fa fa-search"></i></h3>
                 </div>
 
                 <div class="sidebox">
-                    <div id="statistics-table">{% include "problem/statistics-table.html" %}</div>
+                    <div id="statistics-table">
+                        <h3>{{ _('Statistics') }} <i class="fa fa-pie-chart"></i></h3>
+                        <div class="sidebox-content">
+                            <div id="status-graph">
+                                <canvas width="230" height="170"></canvas>
+                            </div>
+                            <div class="total">
+                                {{ _('Total:') }} <span id="total-submission-count">{{ results_total }}</span>
```

Comment:
Does this update? Also I think you do need to have a `trans` block in case there are languages where the total has to come before the label.

Reply:
I don't think we have to worry about this. It is standard practice in to use "X:" to avoid reordering when translating strings, and we have a heavy dependency on stuff like `Time limit:`.

Reply:
And yes, it does update. See `$('#total-submission-count').text(data.total);`.